### ISSUE TITLE
[icetray] make S frame official (Trac #1969)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1961]
If we're going to use S frames in production, they should probably be official.  Add them to icetray so we don't have to use `icetray.I3Frame.Stream('S')`.

I guess the official name should be `icetray.I3Frame.Simulation`?

Migrated from https://code.icecube.wisc.edu/ticket/1969
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:14:38", 
    "description": "If we're going to use S frames in production, they should probably be official.  Add them to icetray so we don't have to use `icetray.I3Frame.Stream('S')`.\n\nI guess the official name should be `icetray.I3Frame.Simulation`?", 
    "reporter": "david.schultz", 
    "cc": "jvansanten, juancarlos", 
    "resolution": "fixed", 
    "_ts": "1550067278746682", 
    "component": "combo core", 
    "summary": "[icetray] make S frame official", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2017-03-21T20:21:46", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20170321T20:37:17: juancarlos commented:

> Maybe icetray.I3Frame.SimluationInfo. We've talked in the past about a potential use of a long time window Simulation frame (t > Q-Frame) at simulation generation.

**Comment by icecube-trac** [comment]
Trac update at 20170321T20:43:46: jvansanten commented:

> The thing about a long frame for holding pre-trigger generated events is that it represents an arbitrary slice of time. Why not make it a T frame? I would also support 0x07 so that `Dump` goes ding every time it sees a frame.

**Comment by icecube-trac** [comment]
Trac update at 20170321T21:06:27: juancarlos commented:

> Well, sure but it seems that the current S-Frame has a very specific function to MuonGun. Do we envision expanding the scope?

**Comment by icecube-trac** [comment]
Trac update at 20170322T06:00:35: kjmeagher commented:

> My understanding is that lepton injector will also use S frames. I also have a branch where nugen uses S frames, which makes calculating weights significantly easier for the user

**Comment by icecube-trac** [comment]
Trac update at 20170322T09:45:32: 
* kjmeagher changed resolution from "" to "fixed"
* kjmeagher changed status from "new" to "closed"
* kjmeagher commented:

> In changeset 2808:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2808"
> Make Simulation frames a first class frame stop. fixes #1961
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190111T20:53:37: nwhitehorn commented:

> In changeset 168791:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="168791"
> Make Simulation frames a first class frame stop. fixes #1961
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:14:38: 
* olivas changed milestone from "September 2017" to ""
* olivas commented:

> Milestone September 2017 deleted
